Client/Frontend/Popover/PopoverController.swift

```diff
@@ -214,6 +262,12 @@ class PopoverController: UIViewController {
             containerView.arrowDirection = direction
         }
         
+        if contentController.extendEdgeIntoArrow {
+            let topInset = containerView.arrowDirection == .up ? PopoverUX.arrowSize.height : 0
+            
+            contentController.additionalSafeAreaInsets = UIEdgeInsets(top: topInset, left: 0, bottom: PopoverUX.arrowSize.height, right: 0)
```

Comment:
Previous logic is changed now, `additonalSafeAreaInsets` are only set for top or bottom depending on where the area is, unlike origin/height where height had to be changed regardless